JetStreamDriver.js

```diff
@@ -62,8 +70,12 @@ if (typeof(URLSearchParams) !== "undefined") {
         customTestList = urlParameters.getAll("test");
     globalThis.testIterationCount = getIntParam(urlParameters, "iterationCount");
     globalThis.testWorstCaseCount = getIntParam(urlParameters, "worstCaseCount");
+    globalThis.prefetchResources = getBoolParam(urlParameters, "prefetchResources", true);
```

Comment:
I would pass `globalThis.prefetchResources` as the default value rather than true. Seems less likely to cause unexpected behavior if someone manually edits a different part of the script.